src/utils.c

```diff
@@ -1788,6 +1788,9 @@ void utils_tidy_path(gchar *filename)
 	const gchar *needle;
 	gboolean preserve_double_backslash = FALSE;
 
+	if (utils_is_uri(filename))
+		return;
```

Comment:
> Sure, I'm aware of the release and the patches are definitely not meant for it.

I didn't mean the release of Geany, I meant the release of RFC 8089 that allows for `file:` to not have the `//`es.  As it was only released Feb 2017 nothing will take advantage of it yet, though as I noted both chrome and firefox accept `file:` URLs without the `//`.